src/core/ui.js

```diff
@@ -56,9 +56,10 @@ const buttons = {};
 sub("start-all", () => document.body.prepend(respecUI), { once: true });
 sub("end-all", () => document.body.prepend(respecUI), { once: true });
 
-const respecPill = hyperHTML`<button id='respec-pill' disabled>ReSpec</button>`;
+const respecPill = hyperHTML`<button id='respec-pill' tabindex="-1" disabled>ReSpec</button>`;
```

Comment:
When checking the preview, I can't seem to tab to the menu at all? also, for whatever reason, it doesn't seem to be focusing? As you seeing the same?

Reply:
Sorry for late reply @marcoscaceres .
As far as I understood from the [comment(5th point)](https://github.com/w3c/respec/issues/884#issuecomment-242375658), we have to make the pill unreachable by pressing tab, And once, The pill is clicked by mouse, focus goes to the pill and things behave as they were earlier. 
Please correct me if I am wrong

Reply:
I don't know - tbh 🥺 We might need to double check with @patrickhlauke if he is around. 

However, while we wait for him to respond, we can add the close button to the dialogs.

Reply:
As I understand, what we need to do is "when a popup opens, make pill and rest of page unreachable", and not make pill unreachable as this PR is doing.

Reply:
> As I understand, what we need to do is "when a popup opens, make pill and rest of page unreachable", and not make pill unreachable as this PR is doing.
@sidvishnoi This seems quite good to me. Shall I go on do this?

Reply:
Also, pressing enter key on the pill, does not open the dialogue box, Shall I add this feature too?

Reply:
Now, the control remains inside the overlay only. If this is right, Can I add the close button too?

Reply:
Also, should I add a feature so that pressing esc closes the dialogue/overay

Reply:
>  Also, should I add a feature so that pressing esc closes the dialogue/overay

This already exists I believe.

Reply:
> This already exists I believe.
Can you please confirm if it exists. because when I am pressing esc, the dialogue is not closing:(

Reply:
Looking at https://w3c.github.io/payment-request/, I can close it by escape already

Reply:
Must be some issue with my system. I am unable to close it here too

Reply:
Just confirming also the "esc" works for me.

Reply:
Oh !  I was asking for pressing Esc on this stage
![Screenshot from 2019-12-31 12-44-34](https://user-images.githubusercontent.com/44131389/71613267-6751f000-2bcb-11ea-96e1-bafa0b5d9e26.png)
Offcourse , it closes when we press esc after selecting one of these options/opening a model

Reply:
Ah, ok - sorry for the misunderstanding. Yes, you are right. It indeed doesn't close there, and probably should.